### ISSUE TITLE
CompressorControlDialog introduces warning

### ISSUE THREAD

**Issue by JohannesLorenz** [LMMS/lmms#5972]
### Bug Summary

Compiler warning. @DouglasDGI can you please try to fix it?

#### Steps to reproduce

Compile LMMS.

#### Expected behavior

No compiler warning.

#### Actual behavior

Compiler warning.

#### Affected LMMS versions

master

#### Logs
<details>
  <summary>Click to expand</summary>
<pre>

[  8%] Building CXX object plugins/Compressor/CMakeFiles/compressor.dir/CompressorControlDialog.cpp.o
/home/johannes/cprogs/lmms/lv2/plugins/Compressor/CompressorControlDialog.cpp: In member function 'virtual void CompressorControlDialog::wheelEvent(QWheelEvent*)':
/home/johannes/cprogs/lmms/lv2/plugins/Compressor/CompressorControlDialog.cpp:644:81: warning: 'int QWheelEvent::delta() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
  644 |  const float dbRangeNew = m_dbRange - copysignf(COMP_GRID_SPACING, event->delta());
      |                                                                                 ^
In file included from /usr/include/qt/QtGui/QMouseEvent:1,
                 from /home/johannes/cprogs/lmms/lv2/plugins/Compressor/CompressorControlDialog.h:30,
                 from /home/johannes/cprogs/lmms/lv2/plugins/Compressor/CompressorControls.h:28,
                 from /home/johannes/cprogs/lmms/lv2/plugins/Compressor/Compressor.h:29,
                 from /home/johannes/cprogs/lmms/lv2/plugins/Compressor/CompressorControlDialog.cpp:25:
/usr/include/qt/QtGui/qevent.h:219:16: note: declared here
  219 |     inline int delta() const  { return qt4D; }
</pre>
</details>

**Comment by JohannesLorenz** [comment]
I just found out there is a similar warning here:

```
/home/johannes/cprogs/lmms/lv2/src/gui/widgets/LcdFloatSpinBox.cpp: In member function 'virtual void LcdFloatSpinBox::wheelEvent(QWheelEvent*)':
/home/johannes/cprogs/lmms/lv2/src/gui/widgets/LcdFloatSpinBox.cpp:180:15: warning: 'int QWheelEvent::x() const' is deprecated: Use position() [-Wdeprecated-declarations]
  180 |  if (event->x() < m_wholeDisplay.width()) { m_intStep = true; }
      |               ^
In file included from /usr/include/qt/QtGui/QMouseEvent:1,
                 from /home/johannes/cprogs/lmms/lv2/src/gui/widgets/LcdFloatSpinBox.cpp:35:
/usr/include/qt/QtGui/qevent.h:229:16: note: declared here
  229 |     inline int x() const { return int(p.x()); }
      |                ^
/home/johannes/cprogs/lmms/lv2/src/gui/widgets/LcdFloatSpinBox.cpp:184:54: warning: 'int QWheelEvent::delta() const' is deprecated: Use angleDelta() [-Wdeprecated-declarations]
  184 |  model()->setValue(model()->value() + ((event->delta() > 0) ? 1 : -1) * getStep());
      |                                                      ^
In file included from /usr/include/qt/QtGui/QMouseEvent:1,
                 from /home/johannes/cprogs/lmms/lv2/src/gui/widgets/LcdFloatSpinBox.cpp:35:
/usr/include/qt/QtGui/qevent.h:219:16: note: declared here
  219 |     inline int delta() const  { return qt4D; }
```

**Comment by DouglasDGI** [comment]
Fixed in #5974.
